### PR TITLE
chore: fix issue template header format

### PR DESCRIPTION
**Pull request by hhyasdf**
# Pull Request Description

## Describe what this PR does / why we need it
Fix issue template header format.

## Does this pull request fix one issue?

None

## Describe how to verify it
Create an issue.

## Special notes for reviews

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -9,19 +9,21 @@ labels: kind/bug
 Please try to use English to describe your issue, or at least provide a snippet of English translation.
 -->
 
-# Bug Report
+Bug Report
+
```
Comment (mars1024) [inline .github/ISSUE_TEMPLATE/bug-report.md:13]: needless line

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -9,19 +9,21 @@ labels: kind/bug
...
+### What happened
 
-## What you expected to happen
+### What you expected to happen
 
-## How to reproduce it (as minimally and precisely as possible)
+### How to reproduce it (as minimally and precisely as possible)
 
-## Anything else we need to know?
+### Anything else we need to know?
```
Comment (mars1024) [inline .github/ISSUE_TEMPLATE/bug-report.md:24]: needless ?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -9,12 +9,14 @@ labels: kind/feature
 Please try to use English to describe your issue, or at least provide a snippet of English translation.
 -->
 
-# Issue Description
+Issue Description
+
```
Comment (mars1024) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: ditto

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -3,19 +3,21 @@
...
+Pull Request Description
 
-## Describe what this PR does / why we need it
+---
 
+### Describe what this PR does / why we need it
 
-## Does this pull request fix one issue?
+
+### Does this pull request fix one issue?
```
Comment (mars1024) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: ditto

File: .github/workflows/check.yml [hunk 5]
```diff
@@ -85,4 +85,5 @@ jobs:
           VALIDATE_MARKDOWN: true
           VALIDATE_MD: true
           DEFAULT_BRANCH: main
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_EXCLUDE: \.github/.*
```
Comment (mars1024) [inline .github/workflows/check.yml:89]: we need a new line here

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (hhyasdf) [reply .github/ISSUE_TEMPLATE/bug-report.md:23]: It exist in Kubernetes issue template too, maybe for extra information?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (hhyasdf) [reply .github/ISSUE_TEMPLATE/bug-report.md]: fixed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (hhyasdf) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (hhyasdf) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: fixed

File: .github/workflows/check.yml (hunk 5 above)
Comment (hhyasdf) [reply .github/workflows/check.yml]: fixed

**Review by mars1024 — APPROVED** [review]
/lgtm

**Pull request merged**